2024-09-26 17:51:12,133 WARNING Not Found: / c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py None
2024-09-26 17:51:12,133 WARNING "GET / HTTP/1.1" 404 2277 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:26:28,735 WARNING Not Found: / c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py None
2024-09-26 18:26:28,735 WARNING "GET / HTTP/1.1" 404 2277 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:28:42,246 WARNING Not Found: / c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py None
2024-09-26 18:28:42,250 WARNING "GET / HTTP/1.1" 404 2277 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:28:44,300 WARNING Not Found: / c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py None
2024-09-26 18:28:44,301 WARNING "GET / HTTP/1.1" 404 2277 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:29:59,202 WARNING Not Found: / c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py None
2024-09-26 18:29:59,202 WARNING "GET / HTTP/1.1" 404 2277 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:30:25,804 INFO "GET /news/ HTTP/1.1" 200 7682 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:30:28,113 INFO "GET /accounts/logout/ HTTP/1.1" 302 0 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:30:28,122 WARNING Not Found: / c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py None
2024-09-26 18:30:28,123 WARNING "GET / HTTP/1.1" 404 2277 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:30:36,365 INFO "GET /admin/ HTTP/1.1" 302 0 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:30:36,385 INFO "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4158 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:30:41,895 INFO "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-09-26 18:30:41,917 INFO "GET /admin/ HTTP/1.1" 200 16220 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:27:37,504 ERROR Internal Server Error: /news/news/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x000002970434F540>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:27:37,547 ERROR "GET /news/news/create/ HTTP/1.1" 500 175291 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:28:04,703 ERROR Internal Server Error: /news/articles/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x00000297045A4400>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:28:04,747 ERROR "GET /news/articles/create/ HTTP/1.1" 500 175330 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:28:23,732 ERROR Internal Server Error: /news/search/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x0000029704691B40>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:28:23,767 ERROR "GET /news/search/ HTTP/1.1" 500 226246 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:28:41,292 ERROR Internal Server Error: /news/subscriptions/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x000002970469C180>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\News-Portal-project\news\views.py", line 170, in subscriptions
    return render(
           ^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:28:41,327 ERROR "GET /news/subscriptions/ HTTP/1.1" 500 209202 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:29:47,289 ERROR Internal Server Error: /news/search/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x00000297045E19C0>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:29:47,315 ERROR "GET /news/search/ HTTP/1.1" 500 226246 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:29:54,756 ERROR Internal Server Error: /news/subscriptions/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x0000029704691F00>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\News-Portal-project\news\views.py", line 170, in subscriptions
    return render(
           ^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:29:54,787 ERROR "GET /news/subscriptions/ HTTP/1.1" 500 209202 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:30:03,715 ERROR Internal Server Error: /news/news/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x00000297046EB480>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:30:03,740 ERROR "GET /news/news/create/ HTTP/1.1" 500 175291 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:30:49,073 ERROR Internal Server Error: /news/1 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x00000297046CB840>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:30:49,103 ERROR "GET /news/1 HTTP/1.1" 500 178715 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:33:47,927 ERROR Internal Server Error: /news/news/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x000002575658BD80>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 4: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:33:47,957 ERROR "GET /news/news/create/ HTTP/1.1" 500 175291 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:35:58,710 ERROR Internal Server Error: /news/1 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x00000262F0111EC0>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:35:58,740 ERROR "GET /news/1 HTTP/1.1" 500 178852 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:35:59,747 ERROR Internal Server Error: /news/1 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?"), <traceback object at 0x00000262F00F9F80>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'trans'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'trans', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:35:59,771 ERROR "GET /news/1 HTTP/1.1" 500 178852 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:43:12,595 ERROR Internal Server Error: /news/news/73/delete/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError('Invalid block tag on line 10: \'"Удаляем"\', expected \'endblock\'. Did you forget to register or load this tag?'), <traceback object at 0x00000262F038D000>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: '"Удаляем"'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 10: '"Удаляем"', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:43:12,618 ERROR "GET /news/news/73/delete/ HTTP/1.1" 500 178546 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:43:42,853 ERROR Internal Server Error: /news/news/73/delete/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError('Invalid block tag on line 10: \'"Удаляем"\', expected \'endblock\'. Did you forget to register or load this tag?'), <traceback object at 0x00000262F03534C0>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: '"Удаляем"'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 10: '"Удаляем"', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:43:42,890 ERROR "GET /news/news/73/delete/ HTTP/1.1" 500 178683 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-03 13:43:43,933 ERROR Internal Server Error: /news/news/73/delete/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError('Invalid block tag on line 10: \'"Удаляем"\', expected \'endblock\'. Did you forget to register or load this tag?'), <traceback object at 0x00000262F0370640>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: '"Удаляем"'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 555, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 10: '"Удаляем"', expected 'endblock'. Did you forget to register or load this tag?
2024-10-03 13:43:43,963 ERROR "GET /news/news/73/delete/ HTTP/1.1" 500 178683 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-04 13:32:06,232 ERROR Internal Server Error: /news/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateDoesNotExist'>, TemplateDoesNotExist('default.html'), <traceback object at 0x00000229D479AC40>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\News-Portal-project\news\views.py", line 34, in get
    return HttpResponse(render(request, 'default.html', context))
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: default.html
2024-10-04 13:32:06,268 ERROR "GET /news/ HTTP/1.1" 500 123703 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-04 16:46:34,545 ERROR Internal Server Error: /news/news/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.utils.datastructures.MultiValueDictKeyError'>, MultiValueDictKeyError('timezone'), <traceback object at 0x0000025332C00580>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 84, in __getitem__
    list_ = super().__getitem__(key)
            ^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'timezone'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\contrib\auth\mixins.py", line 109, in dispatch
    return super().dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\News-Portal-project\news\views.py", line 105, in post
    request.session['django_timezone'] = request.POST['timezone']
                                         ~~~~~~~~~~~~^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 86, in __getitem__
    raise MultiValueDictKeyError(key)
django.utils.datastructures.MultiValueDictKeyError: 'timezone'
2024-10-04 16:46:34,567 ERROR "POST /news/news/create/ HTTP/1.1" 500 95158 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-04 16:47:17,384 ERROR Internal Server Error: /news/news/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.utils.datastructures.MultiValueDictKeyError'>, MultiValueDictKeyError('timezone'), <traceback object at 0x0000018EB7B07480>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 84, in __getitem__
    list_ = super().__getitem__(key)
            ^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'timezone'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\contrib\auth\mixins.py", line 109, in dispatch
    return super().dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\News-Portal-project\news\views.py", line 105, in post
    request.session['django_timezone'] = request.POST['timezone']
                                         ~~~~~~~~~~~~^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 86, in __getitem__
    raise MultiValueDictKeyError(key)
django.utils.datastructures.MultiValueDictKeyError: 'timezone'
2024-10-04 16:47:17,402 ERROR "POST /news/news/create/ HTTP/1.1" 500 95158 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-04 16:47:31,977 ERROR Internal Server Error: /news/articles/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.utils.datastructures.MultiValueDictKeyError'>, MultiValueDictKeyError('timezone'), <traceback object at 0x0000018EB7B593C0>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 84, in __getitem__
    list_ = super().__getitem__(key)
            ^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'timezone'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\contrib\auth\mixins.py", line 109, in dispatch
    return super().dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\News-Portal-project\news\views.py", line 128, in post
    request.session['django_timezone'] = request.POST['timezone']
                                         ~~~~~~~~~~~~^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 86, in __getitem__
    raise MultiValueDictKeyError(key)
django.utils.datastructures.MultiValueDictKeyError: 'timezone'
2024-10-04 16:47:31,992 ERROR "POST /news/articles/create/ HTTP/1.1" 500 95188 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-04 16:50:00,684 ERROR Internal Server Error: /news/articles/53/edit/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'TypeError'>, TypeError("ArticleUpdate.post() got an unexpected keyword argument 'pk'"), <traceback object at 0x0000018EB7AB6480>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\contrib\auth\mixins.py", line 109, in dispatch
    return super().dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: ArticleUpdate.post() got an unexpected keyword argument 'pk'
2024-10-04 16:50:00,695 ERROR "POST /news/articles/53/edit/ HTTP/1.1" 500 83413 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-04 16:52:03,085 ERROR Internal Server Error: /news/news/create/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.utils.datastructures.MultiValueDictKeyError'>, MultiValueDictKeyError('timezone'), <traceback object at 0x000001FB3BF15940>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 84, in __getitem__
    list_ = super().__getitem__(key)
            ^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'timezone'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\contrib\auth\mixins.py", line 109, in dispatch
    return super().dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\News-Portal-project\news\views.py", line 105, in post
    request.session['django_timezone'] = request.POST['timezone']
                                         ~~~~~~~~~~~~^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\datastructures.py", line 86, in __getitem__
    raise MultiValueDictKeyError(key)
django.utils.datastructures.MultiValueDictKeyError: 'timezone'
2024-10-04 16:52:03,101 ERROR "POST /news/news/create/ HTTP/1.1" 500 95146 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
2024-10-04 16:59:28,741 ERROR Internal Server Error: /news/search/ c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\utils\log.py (<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Could not parse some characters: |{% trans 'post.heading'%}||censor"), <traceback object at 0x0000017C5E8EFBC0>)
Traceback (most recent call last):
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\defaulttags.py", line 961, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\defaulttags.py", line 861, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\template\base.py", line 670, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse some characters: |{% trans 'post.heading'%}||censor
2024-10-04 16:59:28,784 ERROR "GET /news/search/ HTTP/1.1" 500 267174 c:\Users\Кристина\Desktop\общая\phyton\VS_Projects\Module D\.venv\Lib\site-packages\django\core\servers\basehttp.py None
